package tka_may12_ExceptionHandling;

public class C5_ExpDev {
	public static void main(String[] args) {
		
		System.out.println("====Welcome====");
		
		try {
			String s = null;
			s.toUpperCase();
			
			int a = 1 / 0;				
		}
//		catch (ArithmeticException e) {
//			e.printStackTrace();
//		}
		catch (NullPointerException e) {
			e.printStackTrace();
		}
		catch (Exception e) {
//		catch (Exception e) {
			e.printStackTrace();
		}
		
		System.out.println("thanks..");
	}
}


// thanks... is not printing in above code because string exception was not handled. We are handling ArithmeticException but before it we have NullPointerException so we should handle NullPointerException or Exception first.
